diff -uraN gamescope-3.16.25/src/meson.build omv-gamescope-3.16.25/src/meson.build
--- gamescope-3.16.25/src/meson.build	2026-07-21 12:58:10.000000000 +0200
+++ omv-gamescope-3.16.25/src/meson.build	2026-08-02 17:11:30.598623955 +0200
@@ -25,8 +25,8 @@
 udev_dep = dependency('libudev')
 
 wlroots_dep = dependency(
-  'wlroots-0.19',
-  version: ['>= 0.19.0', '< 0.20.0'],
+  'wlroots-0.20',
+  version: ['>= 0.20.0', '< 0.21.0'],
   fallback: 'wlroots',
   default_options: ['default_library=static', 'examples=false', 'xwayland=enabled', 'backends=libinput', 'renderers=[]', 'allocators=[]', 'session=enabled'],
 )
