diff -uraN gamescope-3.16.25/meson.build omv-gamescope-3.16.25/meson.build
--- gamescope-3.16.25/meson.build	2026-07-21 12:58:10.000000000 +0200
+++ omv-gamescope-3.16.25/meson.build	2026-08-02 17:09:58.932006400 +0200
@@ -6,12 +6,12 @@
   default_options: [
     'cpp_std=c++20',
     'warning_level=2',
-    'force_fallback_for=libliftoff,vkroots',
+    'force_fallback_for=vkroots',
   ],
 )
 
 fallbacks = get_option('force_fallback_for')
-if not (fallbacks.contains('libliftoff') and fallbacks.contains('vkroots'))
+if not (fallbacks.contains('vkroots'))
   error('!!!"force_fallback_for" is missing entries!!!\n\tPlease do not remove entries from force_fallback_for if you are packaging the project.\n\tWe pull in these projects at specific commits/forks/builds for a reason.\n\tIf you are not packaging, remove this line to continue.')
 endif
